.github/workflows/build-translation-source.yml

```diff
@@ -62,7 +50,3 @@ jobs:
           upload_sources: true
           upload_translations: true
           config: crowdin.yml
-        env:
-          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
-          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
why does moving the env to a global scope resolve the issue? Looks like this exposes secrets globally (e.g. to pip install) for no good reason?